Marcio/registrar revamp (#2052)
cloudflare/cloudflare-docs: products/registrar/src/content/faq.md

If a year is not added to your registration, you have effectively paid twice for the same added year. Per ICANN rules, you are entitled to request a refund at your previous registrar.

## What happens to my name servers when I transfer my domain to Cloudflare?

Cloudflare Registrar only supports transfers of domains that are active on Cloudflare. Active domains on Cloudflare use nameservers assigned by Cloudflare. When you transfer your registration, your nameservers will not be modified.

## How can I see the status of my domain transfer?

Once you initiate a domain transfer, your previous registrar has five days to release the domain. In most cases, they will send you an email to confirm you want to transfer. If you actively acknowledge that email (through a link or the registrar's dashboard), they can process it immediately.

> To see the progress of your transfer, from the Cloudflare dashboard select the **Domain Registration** tab or the **Transfer Domains** button, both in your account home page. Once there, you will see a list of domain transfers that are in progress. To accelerate the process, be sure to check with your old registrar how you can approve the transfer out.

Reviewer: ```suggestion
To see the progress of your transfer, from the Cloudflare dashboard > go to **Domain Registration** or the **Transfer Domains** button, both in your account home page. Once there, you will see a list of domain transfers that are in progress. To accelerate the process, be sure to check with your old registrar how you can approve the transfer out.
```
Are we talking about the Account Home? Or the Overview? Because instructions are unclear. 

Domain registration is in **Overview** > **Domain Registration**. 

I can't find **Transfer Domains**. Are you referring to the *Transfer out** button in **Domain Registration** > **Manage Domain** ? 

<img width="1865" alt="account home" src="https://user-images.githubusercontent.com/70746074/130072709-e8fa8d65-893b-4048-95b1-7bde9c25a13a.png">
<img width="1514" alt="overview" src="https://user-images.githubusercontent.com/70746074/130072727-da4f7a68-1ecc-4166-85ae-d97052c55e16.png">

Once successful, you will receive an email from Cloudflare and be able to manage the domain in the Overview tab for that site.

## What happens if I entered the wrong auth code?

> If you input the incorrect authentication code, please return to the **Domain Registration** page or the **Overview** app for your site. You can use the available input field to reenter your authentication code.

Author: It depends. It's `auth` in some places, `authorization` in others

> If you input the incorrect auth code (also referred to as authentication code), please return to the **Domain Registration** page or the **Overview** app for your site. You can use the available input field to reenter your authentication code.

Reviewer: ```suggestion
If you input the incorrect auth code (also referred to as authentication code and authorization code), return to the **Domain Registration** page or the **Overview** for your site. You can use the available input field to reenter your authentication code.
```

## Why did my transfer fail?

Domain transfers sometimes fail. We are here to help you address what failed so that you can complete the transfer to Cloudflare.

> First, make sure that you entered the auth code for that domain. If you need to reenter it, you can do so from the transfer launch page or the overview page for that site.

Reviewer: Do you need bolds here? 

```suggestion
First, make sure that you entered the auth code for that domain. If you need to reenter it, you can do so from the **Transfer Launch** page or the **Overview** for that site.
```

> Second, you can reattempt the transfer from the transfer launch page.

Reviewer: Same thing here.

Author: What do you mean? I'm not sure this is showing up to me in the same order you wrote this comment :)

Reviewer: Ah sorry, this was meant for the descriptions of the directions. Is it Account Home > Overview > Transfer? Because I don't believe such a path exists right now on the dash.

Author: I have to check this, because I believe some directions / breadcrumbs are only available if you've just transferred a domain to Cloudflare or are in that process. Otherwise you won't see the option in the Dashboard.

> If you cannot seem to resolve the issue, please open a support ticket and we will help investigate.

Reviewer: ```suggestion
If you cannot resolve the issue, open a support ticket or contact your account team.
```

## Why am I not allowed to transfer my domain?

> ICANN prohibits domain transfers within 60 days of a change to the WHOIS data or registrar of a domain. If you modified your contact information, transferred registrars, or registered your domain in the last 60 days, we will be unable to process your transfer immediately.

Reviewer: ```suggestion
ICANN prohibits domain transfers within 60 days of a change to the WHOIS data or registrar of a domain. If you modified your contact information, transferred registrars, or registered your domain in the last 60 days, Cloudflare will be unable to process your transfer immediately.
```

> You can leave the domain `In Progress` and we will attempt to process the transfer until the 60-day window passes.

Reviewer: ```suggestion
You can leave the domain **In Progress** and Cloudflare will attempt to process the transfer until the 60-day window passes.
```

## Why am I not able to start a transfer?

If you have an <a href='https://support.cloudflare.com/hc/articles/203471284#h_1l0KGygoBX9QYjNrhAcHjg'>unverified email address</a>, you might experience issues when initiating a domain transfer.

## Premium Domains

Cloudflare does not currently support premium domains. Some registries designate a domain name as “premium” and charge higher wholesale rates for these domains.

## What Happens When a Domain Expires?

> Cloudflare currently offers a 40 day grace period for most TLDs. During this period you may renew/extend the domain at any time from within the dashboard but no further auto-renew attempts will be made. For the first 30 days of the grace period, the domain will continue to resolve as normal. On the 30th day after the expiration date, the domain will be suspended and a parked suspension page will be displayed. You may still renew the domain at any time during this suspension period. On the 40th day, the domain will enter the Redemption Period and will no longer resolve to any web page. The redemption period lasts for 30 days.  During this time, it may be possible to restore and renew the domain. A restore fee may apply in addition to the renewal fee. Please contact support for assistance. At the end of the 30 day redemption period, the domain will be placed in pending delete status for a period of five days, after which it will be released and made available for re-registration. The domain cannot be restored or renewed during this period.

Reviewer: ```suggestion
Cloudflare currently offers a 40 day grace period for most top-level domains (TLDs). During this period you may renew/extend the domain at any time from within the dashboard but no further auto-renew attempts will be made. For the first 30 days of the grace period, the domain will continue to resolve as normal. On the 30th day after the expiration date, the domain will be suspended and a parked suspension page will be displayed. You may still renew the domain at any time during this suspension period. On the 40th day, the domain will enter the Redemption Period and will no longer resolve to any web page. The redemption period lasts for 30 days.  During this time, it may be possible to restore and renew the domain. A restore fee may apply in addition to the renewal fee. Contact your account team for assistance. At the end of the 30 day redemption period, the domain will be placed in pending delete status for a period of five days, after which it will be released and made available for re-registration. The domain cannot be restored or renewed during this period.
```
